packages/expo-camera/CHANGELOG.md

```diff
@@ -7,12 +7,13 @@
 ### 🎉 New features
 
 - Delete `prop-types` in favor of TypeScript. ([#8680](https://github.com/expo/expo/pull/8680) by [@EvanBacon](https://github.com/EvanBacon))
+- [camera] Directly import `createElement` from `react-native-web` for RNW v12 support. ([#8773](https://github.com/expo/expo/pull/8773) by [@EvanBacon](https://github.com/EvanBacon))
 
 ### 🐛 Bug fixes
 
 ## 8.3.1 — 2020-05-29
 
-*This version does not introduce any user-facing changes.*
+_This version does not introduce any user-facing changes._
```

Comment:
Same ^^^

Reply:
We do, see here https://github.com/expo/expo/pull/8774

Reply:
Oh, I see. Thanks ;)